tools/clang/test/CodeGenSPIRV/intrinsics.vkrawbufferload.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -T ps_6_0 -E main
+
+// CHECK: OpCapability PhysicalStorageBufferAddresses
+// CHECK: OpExtension "SPV_KHR_physical_storage_buffer"
+// CHECK: OpMemoryModel PhysicalStorageBuffer64 GLSL450
+
+uint64_t Address;
+float4 main() : SV_Target0 {
+  // CHECK: OpTypePointer PhysicalStorageBuffer %uint
+  // CHECK: OpBitcast %_ptr_PhysicalStorageBuffer_uint %24
```

Comment:
We avoid relying on the numerical result-ids (`%24`, `%26`, `%27`, etc) in tests since they can change and don't affect the correctness of the test. Switch these to use pattern matching ([example](https://github.com/microsoft/DirectXShaderCompiler/blob/e4ddf78452be1354d16d3a8cddb73eb6e252a1ca/tools/clang/test/CodeGenSPIRV/binary-op.assign.opaque.array.hlsl#L27-L30)).

Reply:
Done.